app/src/main/java/ryey/easer/skills/operation/hotspot/HotspotHelper.java

```diff
@@ -63,6 +106,24 @@ WifiConfiguration getApConfiguration() throws NoSuchMethodException, InvocationT
     boolean setApStatus(WifiConfiguration netConfig, boolean enable) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
         Method setWifiApMethod = wifiManager.getClass().getMethod("setWifiApEnabled", WifiConfiguration.class, boolean.class);
         boolean apStatus = (boolean) setWifiApMethod.invoke(wifiManager, netConfig, enable);
+        if (!apStatus) {
```

Comment:
Could you move the following code to a new function, and call it here?

Reply:
ok